### ISSUE TITLE
Not able to add EnableTestFixtureOrdering to AssemblyInfo.cs file

### ISSUE THREAD

**Issue by AutomationSifu** [Sebazzz/NUnitTestOrdering#6]
When adding the [assembly: EnableTestFixtureOrdering] line to the AssemblyInfo.cs file the using NUnitOrderedTest directive is not being pulled into the file.

I have downloaded the beta release of the NUnitTestOrdering package via NuGet.

I've listed my version numbers below:

NUnit - 3.9.0
NUnitTestOrdering - v1.0.0-beta1

Package Manager Console output below:

I've excluded the ProjectName field.

PM> get-package

Id                                                        Versions                                                                                                                                                                                 
--                                                         --------                                                                                                                                                                             
NUnit                                                    {3.9.0}                                                                                                                                                                                
NUnitTestOrdering                               {1.0.0-beta1}                                                                                                                                                                       
Selenium.Support                                 {3.10.0}                                                                                                                                                                               
Selenium.WebDriver                             {3.10.0}                                                                                                                                                                               
Selenium.WebDriver.ChromeDriver      {2.36.0}                                

Thanks!

**Comment by Sebazzz** [comment]
Please submit a minimal repro project.

**Comment by AutomationSifu** [comment]
I'm very new to Github.  For your request, are you just asking to upload a very small project to Github or are you saying something different?

**Comment by Sebazzz** [comment]
Yes, the minimum amount of code needed to reproduce the problem. You can add it as an attachment in your comment, or upload it as a repository on Github.

**Comment by AutomationSifu** [comment]
I created a new project with the least amount of libraries/files to reproduce the problem.  After installing the NUnit and NUnitTestOrdering packages I can reproduce the issue.  One thing I did notice is after installing the NUnitTestOrdering package via Nuget a file automatically created.  The file name is EnableTestOrdering.cs.  The file includes the following two lines by default:  using NUnitTestOrdering.FixtureOrdering; and [assembly:EnableTestOrdering].  The using directive still shows a red line to indicate the directive is not resolved.

**Comment by AutomationSifu** [comment]
Hi Sebazz,

I hope all is well!  Checking in to see if there is any additional feedback.
Thanks!

**Comment by Sebazzz** [comment]
Do you have any compiler warnings? On which platform (.NET version) are you installing?

Met vriendelijke groet,
Sebastiaan Dammann

________________________________
Van: AutomationSifu <notifications@github.com>
Verzonden: maandag, augustus 27, 2018 8:02 PM
Aan: Sebazzz/NUnitTestOrdering
CC: Sebastiaan Dammann; Comment
Onderwerp: Re: [Sebazzz/NUnitTestOrdering] Not able to add EnableTestFixtureOrdering to AssemblyInfo.cs file (#6)


Hi Sebazz,

I hope all is well! Checking in to see if there is any additional feedback.
Thanks!

**Comment by AutomationSifu** [comment]
**Version numbers:**

<package id="NUnit" version="3.7.1" targetFramework="net452" />
<package id="NUnitTestOrdering" version="1.0.0-alpha5" targetFramework="net452" />


**Compiler errors**

Severity	Code	Description	Project	File	Line	Suppression State

Error	CS0246	The type or namespace name 'EnableTestOrdering' could not be found (are you missing a using directive or an assembly reference?)	sampleProject2	C:\...\EnableTestOrdering.cs	3	Active


Error	CS0246	The type or namespace name 'EnableTestOrderingAttribute' could not be found (are you missing a using directive or an assembly reference?)	sampleProject2	C:\...\AssemblyInfo.cs	37	Active


Error	CS0246	The type or namespace name 'EnableTestOrdering' could not be found (are you missing a using directive or an assembly reference?)	sampleProject2	C:\...\AssemblyInfo.cs	37	Active


Error	CS0246	The type or namespace name 'NUnitTestOrdering' could not be found (are you missing a using directive or an assembly reference?)	sampleProject2	C:\...\EnableTestOrdering.cs	1	Active


Error	CS0246	The type or namespace name 'EnableTestOrderingAttribute' could not be found (are you missing a using directive or an assembly reference?)	sampleProject2	C:\...\EnableTestOrdering.cs	3	Active

**Comment by AutomationSifu** [comment]
Hi Sebazzz,

I did some more testing on this item.  I created another project on my personal machine and was able to get the using NUnitTestOrdering directive to show up in the project by using version 4.6.1 of the .Net framework.  The one issue I'm still having is getting the following assembly reference to resolve in the AssemblyInfo.cs file:  [assembly: EnableTestFixtureOrdering].

 I have a few quick questions to help make things more clear:

1.  Is there a minimum .NET version to be used with the NUnitTestOrdering library?

2.  The EnableTestOrdering.cs file is created by default, is this needed for the project?

3.  Should the [assembly: EnableTestFixtureOrdering] reference be added to the AssemblyInfo.cs file?

Thanks!
